airbyte-commons-worker/src/main/java/io/airbyte/workers/helper/CatalogClientConverters.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.workers.helper;
+
+import io.airbyte.commons.enums.Enums;
+import io.airbyte.commons.text.Names;
+import io.airbyte.protocol.models.AirbyteStream;
+import java.util.stream.Collectors;
+
+/**
+ * Utilities to convert Catalog protocol to Catalog API client.
+ */
+public class CatalogClientConverters {
```

Comment:
Probably worth adding some explicit unit tests for this class, especially since catalog conversions can be error prone and finicky!